lang/Polski.json

```
@@ -108,53 +108,53 @@
         },
         "help": {
             "links": {
                 "name": "Pomocne linki",
-                "inventory": "Zobacz ekwipunek na Twitchu",
+                "inventory": "Zobacz swój ekwipunek na Twitchu",
                 "campaigns": "Zobacz wszystkie kampanie na Twitchu"
             },
             "how_it_works": "Jak to działa?",
-            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o zwiększeniu lub zmniejszeniu ilości strumieni, lub aktualizuje aktualną ilości widzów.",
+            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o zmianie statusu kanałów, oraz aktualizuje aktualną ilości widzów.",
             "getting_started": "Pierwsze kroki",
-            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię tylko dropienie wszystkiego, odznacz „Tylko priorytet” i naciśnij na „Przeładuj”.\n• Jeśli chcesz wydropić konkretne gry, użyj listy „Priorytet”, aby ustawić uporządkowaną listę wybranych przez siebie gier. Gry z górnej części listy będą dropione jako pierwsze, przed tymi z dołu listy.\n• Zaznacz opcję „Tylko priorytet”, aby wykluczyć gry z dropienia, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, by wskazać aplikacji kampanie gier, które nigdy nie powinny być dropione.\n• Zmiana wartości lub statusu „Tylko priorytet” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
+            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```

```suggestion
            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```
Just a single comma too much in the third bullet point, without it it should be good.

---

There should be a comma before `"co"` so I suggest removing the next comma. The comma before `"co"` is more important than the one before `"odznacz"`. If I add a comma before `"co"` then the next one is redundant.

`Jeśli interesuje cię dropienie wszystkiego, co możliwe odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.
`

---

```suggestion
            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```